python/ray/tests/BUILD

```diff
@@ -9,105 +9,112 @@ SRCS = [] + select({
+        "test_global_gc.py",
+        "test_iter.py",
+        "test_joblib.py",
+        "test_resource_demand_scheduler.py",
+    ],
+    tags = [
+        "exclusive",
+        "medium_size_python_tests_a_to_j",
+    ],
+    deps = ["//:ray_lib"],
 )
 
 py_test_module_list(
-  files = [
-    "test_memory_limits.py",
-    "test_memory_scheduling.py",
-    "test_metrics.py",
-    "test_multi_node_2.py",
-    "test_multinode_failures_2.py",
-    "test_multinode_failures.py",
-    "test_multi_node.py",
-    "test_multiprocessing.py",
-    "test_object_manager.py",
-    "test_output.py",
-    "test_placement_group.py",
-    "test_reconstruction.py",
-    "test_reference_counting_2.py",
-    "test_reference_counting.py",
-    "test_stress.py",
-    "test_stress_sharded.py",
-    "test_unreconstructable_errors.py",
-    "test_tensorflow.py",
-    "test_object_spilling.py",
-  ],
-  size = "medium",
-  extra_srcs = SRCS,
-  tags = ["exclusive", "medium_size_python_tests_k_to_z"],
-  deps = ["//:ray_lib"],
+    size = "medium",
+    extra_srcs = SRCS,
+    files = [
+        "test_memory_limits.py",
+        "test_memory_scheduling.py",
+        "test_metrics.py",
+        "test_multi_node.py",
+        "test_multi_node_2.py",
+        "test_multinode_failures.py",
+        "test_multinode_failures_2.py",
+        "test_multiprocessing.py",
+        "test_object_manager.py",
+        "test_object_spilling.py",
+        "test_output.py",
+        "test_placement_group.py",
+        "test_reconstruction.py",
+        "test_reference_counting.py",
+        "test_reference_counting_2.py",
+        "test_stress.py",
+        "test_stress_sharded.py",
+        "test_tensorflow.py",
+        "test_unreconstructable_errors.py",
+    ],
+    tags = [
+        "exclusive",
+        "medium_size_python_tests_k_to_z",
+    ],
+    deps = ["//:ray_lib"],
 )
 
 py_test_module_list(
-  files = [
-    "test_actor_pool.py",
-    "test_args.py",
-    "test_asyncio.py",
-    "test_autoscaler.py",
-    "test_autoscaler_yaml.py",
-    "test_component_failures.py",
-    "test_command_runner.py",
-    "test_coordinator_server.py",
-    "test_dask_scheduler.py",
-    "test_debug_tools.py",
-    "test_global_state.py",
-    "test_job.py",
-    "test_memstat.py",
-    "test_metrics_agent.py",
-    "test_microbenchmarks.py",
-    "test_mini.py",
-    "test_multi_tenancy.py",
-    "test_node_manager.py",
-    "test_numba.py",
-    "test_ray_init.py",
-    "test_serialization.py",
-    "test_tempfile.py",
-    "test_webui.py",
-  ],
-  size = "small",
-  extra_srcs = SRCS,
-  tags = ["exclusive"],
-  deps = ["//:ray_lib"],
+    size = "small",
+    extra_srcs = SRCS,
+    files = [
+        "test_actor_pool.py",
+        "test_args.py",
+        "test_asyncio.py",
+        "test_autoscaler.py",
+        "test_autoscaler_yaml.py",
+        "test_command_runner.py",
+        "test_component_failures.py",
+        "test_coordinator_server.py",
+        "test_dask_callback.py",
```

Comment:
This file is the only actualy change to this file, the rest is due to my Bazel [buildifier](https://github.com/bazelbuild/buildtools/blob/master/buildifier/README.md) autoformatting [plugin](https://github.com/google/vim-codefmt). I was going to remove these formatting changes, but I wanted to gauge the interest in adding buildifier to Ray's formatting suite. We use it internally (at Descartes Labs) and really like it.